lib/src/common/point.dart

```diff
@@ -0,0 +1,27 @@
+part of tiled;
+
+/// <point>
```

Comment:
I think you can drop these since the name is just below, or are they part of dartdoc?

Reply:
this is just showing how the xml tag looks like. it's copied from the tile docs

Reply:
Ah, that makes sense.